### PR TITLE
externalizing card

### PR DESCRIPTION
**Pull request by mathewsunaal**
- removed CARD docker build and source code from repository
- externalized CARD as an image
- updated docker compose files

### CODE REVIEW

File: models/hovernet/Dockerfile_hovernet [hunk 1]
```diff
@@ -20,10 +9,11 @@ COPY . .
 ENV DEBIAN_FRONTEND=noninteractive
 
 # install libraries
-RUN apt-get update && apt-get install -y wget
+RUN apt-get clean && apt-get update
+RUN apt-get install -y wget
 RUN apt-get install -y git-all
 RUN apt install -y build-essential checkinstall libreadline-gplv2-dev libncursesw5-dev libssl-dev libsqlite3-dev tk-dev libgdbm-dev libc6-dev libbz2-dev
-RUN apt update && apt install -y openslide-tools libncurses-dev libgdbm-dev libz-dev tk-dev libsqlite3-dev libreadline-dev liblzma-dev libffi-dev libssl-dev
```
Comment (blankachamradova) [inline models/hovernet/Dockerfile_hovernet:26]: does it find all the libraries even without the update?

File: Dockerfile [hunk 2]
```diff
@@ -1,14 +1,5 @@
 FROM nvidia/cuda:11.8.0-base-ubuntu20.04
 
-ARG NO_PROXY
```
Comment (blankachamradova) [inline Dockerfile:3]: let's remove this from the readme if we are disabling this here (https://github.com/Sanofi-Public/spatialone-pipeline/blob/master/docs_md/cardhovernet_tutorial.md#3-set-up-the-environment)

Comment (AlbertPlaPlanas) [reply Dockerfile:3]: Done.

File: models/hovernet/Dockerfile_hovernet (hunk 1 above)
Comment (mathewsunaal) [reply models/hovernet/Dockerfile_hovernet:26]: Yes it should since there is an apt-update in the first line of this install block.

**Review by AlbertPlaPlanas — APPROVED** [review]
Approving based on @FabienPertuy approval on internal repo

**Pull request merged**